src/components/plan/PlanMergeReview.svelte

```diff
@@ -293,19 +298,21 @@
   }
 
   async function onApproveChanges() {
-    if (initialMergeRequest !== null) {
-      const success = await effects.planMergeCommit(
-        initialMergeRequest.id,
-        initialMergeRequest.plan_snapshot_supplying_changes.plan,
-        initialMergeRequest.plan_receiving_changes,
-        user,
-      );
-      if (success) {
-        $planReadOnlyMergeRequest = false;
-        userInitiatedMergeRequestResolution = true;
-        goto(`${base}/plans/${initialPlan.id}`);
+    await showPlanBranchMergeDerivationGroupMessageModal(sourcePlanName, targetPlanName).then(async () => {
```

Comment:
Will this appear for all plan merges regardless of whether or not the plan has any events?

Reply:
As of now, yes.

Reply:
As of [this commit](https://github.com/NASA-AMMOS/aerie-ui/pull/1396/commits), it only shows if the plan has derivation groups associated.